### DISCUSSION TITLE
nice collection

### DISCUSSION THREAD

**Discussion by aulneau** [antonilol/btc_stuff#5] [General]
hi there! spotted your repo as i've been digging into more bitcoin related stuff, just wanted to pass along this new library -> https://github.com/paulmillr/micro-btc-signer would be cool to get your eyes on it as you've got a good amount of experience with bitcoinjs-lib it seems

**Comment by antonilol** [comment]
> this new library -> https://github.com/paulmillr/micro-btc-signer

looks good! however, you are linking a bitcoin utility _library_ (like you could use in a project as npm dep). This repo is a bunch of script i made when learning and playing around with bitcoin scripts and other tech

> would be cool to get your eyes on it as you've got a good amount of experience with bitcoinjs-lib it seems

do you (or the author of micro btc signer) want my help on something bitcoinjs-lib related? (micro btc signer doesnt have it as npm dep...)
